integration/integration_test.go

```diff
@@ -18,6 +18,7 @@ import (
 
 	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
 	schemapb "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/schema"
+	"github.com/TBD54566975/ftl/integration"
```

Comment:
Ah I should have mentioned I think in cases like this it's fine to use a "dot" import eg.

```
. "github.com/TBD54566975/ftl/integration"
```

This brings all of the symbols from that package into the current namespace, meaning you don't have to prefix every symbol with the package name.

Reply:
oh neat! I didn't know about this either. Thanks!